config.json

```
@@ -527,6 +527,24 @@
                 "math",
                 "integers"
             ]
+        },
+        {
+            "slug": "minesweeper",
+            "uuid": "9a47bffd-4dd2-48ce-b47b-753271305966",
+            "core": false,
+            "unlocked_by":"nucleotide-count",
```

```suggestion
            "unlocked_by": "pangram",
```

`nucleotide-count` already unlocks a lot of exercises and arrays haven't been introduced to the core progression path yet, so let's move it to be unlocked by pangram.